### ISSUE TITLE
Tapping Photo Picker on iPad crashes the app

### ISSUE THREAD

**Issue by AnitaPita** [intercom/intercom-ios#153]
Hi guys, I've noticed this issue when tapping on the Photo icon in the chat bar of Intercom. The app is locked to Landscape mode and must remain so. I'm running this on an iPad Pro.

## Version info
  - Intercom for iOS version: 3.0.3
  - iOS version: 9.3.2

## Expected behavior
The app should open the camera roll to select a photo for submission.

## Actual behavior
The app remains on a loading screen for the camera roll. XCode shows a crash.

## Steps to reproduce
 1. Open the intercom chat window
 2. Tap the photo icon on the chat bar

## Logs
```
2016-07-27 09:04:27.633 Shopify POS[90873:2624512] *** Terminating app due to uncaught exception 'UIApplicationInvalidInterfaceOrientation', reason: 'Supported orientations has no common orientation with the application, and [PUUIAlbumListViewController shouldAutorotate] is returning YES'
*** First throw call stack:
(
	0   CoreFoundation                      0x070a6494 __exceptionPreprocess + 180
	1   libobjc.A.dylib                     0x0686ee02 objc_exception_throw + 50
	2   CoreFoundation                      0x070a63bd +[NSException raise:format:] + 141
	3   UIKit                               0x04f339d3 -[UIViewController __supportedInterfaceOrientations] + 1000
	4   UIKit                               0x04f33faa -[UIViewController __withSupportedInterfaceOrientation:apply:] + 49
	5   UIKit                               0x04f348f1 -[UIViewController setInterfaceOrientation:] + 147
	6   UIKit                               0x04f1afd9 -[UIViewController viewDidMoveToWindow:shouldAppearOrDisappear:] + 1580
	7   UIKit                               0x04e0b6fd -[UIView(Internal) _didMoveFromWindow:toWindow:] + 1854
	8   UIKit                               0x04e0b399 -[UIView(Internal) _didMoveFromWindow:toWindow:] + 986
	9   UIKit                               0x04e0b399 -[UIView(Internal) _didMoveFromWindow:toWindow:] + 986
	10  UIKit                               0x04e0b399 -[UIView(Internal) _didMoveFromWindow:toWindow:] + 986
	11  UIKit                               0x04dfede7 __45-[UIView(Hierarchy) _postMovedFromSuperview:]_block_invoke + 180
	12  Foundation                          0x037e1440 -[NSISEngine withBehaviors:performModifications:] + 150
	13  Foundation                          0x037e491c -[NSISEngine withAutomaticOptimizationDisabled:] + 48
	14  UIKit                               0x04dfece4 -[UIView(Hierarchy) _postMovedFromSuperview:] + 521
	15  UIKit                               0x04e0e7f1 -[UIView(Internal) _addSubview:positioned:relativeTo:] + 2367
	16  UIKit                               0x04dfcd11 -[UIView(Hierarchy) insertSubview:belowSubview:] + 68
	17  Intercom                            0x04a1eabb -[PodIntercom_ITBUploadPickerController viewDidAppear:] + 473
	18  UIKit                               0x04f1866e -[UIViewController _setViewAppearState:isAnimating:] + 774
	19  UIKit                               0x04f18ded -[UIViewController __viewDidAppear:] + 166
	20  UIKit                               0x04f190c6 -[UIViewController _endAppearanceTransition:] + 280
	21  UIKit                               0x04f3be76 -[UIViewController(UIContainerViewControllerProtectedMethods) endAppearanceTransition] + 41
	22  UIKit                               0x04edcb70 -[UIPresentationController transitionDidFinish:] + 939
	23  UIKit                               0x04ee09ee __56-[UIPresentationController runTransitionForCurrentState]_block_invoke_2 + 224
	24  UIKit                               0x058b6c5f -[_UIViewControllerTransitionContext completeTransition:] + 118
	25  UIKit                               0x058e1b34 -[UIViewControllerBuiltinTransitionViewAnimator transitionViewDidComplete:fromView:toView:removeFromView:] + 74
	26  UIKit                               0x04ed948a -[UITransitionView notifyDidCompleteTransition:] + 280
	27  UIKit                               0x04ed9176 -[UITransitionView _didCompleteTransition:] + 1550
	28  UIKit                               0x04edbdb1 -[UITransitionView _transitionDidStop:finished:] + 121
	29  UIKit                               0x04de008f -[UIViewAnimationState sendDelegateAnimationDidStop:finished:] + 247
	30  UIKit                               0x04de0485 -[UIViewAnimationState animationDidStop:finished:] + 90
	31  QuartzCore                          0x0485368b _ZN2CA5Layer23run_animation_callbacksEPv + 305
	32  libdispatch.dylib                   0x081409cd _dispatch_client_callout + 14
	33  libdispatch.dylib                   0x08125f7c _dispatch_main_queue_callback_4CF + 910
	34  CoreFoundation                      0x06ff01be __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 14
	35  CoreFoundation                      0x06fae434 __CFRunLoopRun + 2356
	36  CoreFoundation                      0x06fad846 CFRunLoopRunSpecific + 470
	37  CoreFoundation                      0x06fad65b CFRunLoopRunInMode + 123
	38  GraphicsServices                    0x0b13b664 GSEventRunModal + 192
	39  GraphicsServices                    0x0b13b4a1 GSEventRun + 104
	40  UIKit                               0x04d41eb9 UIApplicationMain + 160
	41  Shopify POS                         0x0004db62 main + 178
	42  libdyld.dylib                       0x0816aa25 start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by jtreanor** [comment]
Thanks for reporting this issue @AnitaPita. We will fix this in the next release.

The issue here is that `UIImagePickerController` does not natively support landscape. I'll look into the best way to solve this.

**Comment by jtreanor** [comment]
@AnitaPita We have added a short term workaround for the crash in [3.0.4](https://github.com/intercom/intercom-ios/releases/tag/3.0.4).
The workaround is that the upload button will be hidden on devices which it will crash on.

The longer term solution will be to replace `UIImagePickerController` which I had added to our tasks internally. In the meantime I was keen to prevent this crash from occurring.
